src/linked_list.py

```diff
@@ -0,0 +1,127 @@
+"""Link list implementation."""
+import sys
+
+
+class Node(object):
+    """Define node object."""
+
+    def __init__(self, value, next_up):
+        """Instantiate an object with a given value."""
+        self.value = value
+        self.next = next_up
+
+
+class Linked_List(object):
```

Comment:
This shouldn't have an underscore. Class object names should be camel case with the first letter being capitalized